src/main/AndroidManifest.xml

```diff
@@ -39,7 +39,6 @@
     <activity android:name="LockScreen"
         android:screenOrientation="portrait"/>
     <activity android:name="EmbeddedBrowserActivity"
-        android:screenOrientation="portrait"
```

Comment:
Good thinking, but I think this will also affect phone users. For this issue we specifically want phone users to be locked to portrait orientation, but allow tablet users to switch orientation, or possibly even be locked to landscape. Do you know of any way to change the orientation declaration depending on the screen size or type of device?

Reply:
Modified the code now and